Tests/Fixtures/UrlLoadBalancerFixtures.swift

```diff
@@ -0,0 +1,270 @@
+//
+//  Copyright (c) 2020-2022 MobileCoin. All rights reserved.
+//
+import LibMobileCoin
+@testable import MobileCoin
+import XCTest
+
+// chose to avoid the extension on the Generic UrlLoadBalancer<> as the
+// use of the fixture would be obtrusive
+struct UrlLoadBalancerFixtures {
+    let invalidFogUrlBalancer: SequentialUrlLoadBalancer<FogUrl>
+    let invalidConsensusUrlBalancer: SequentialUrlLoadBalancer<ConsensusUrl>
+
+    init() throws {
+        invalidFogUrlBalancer = try Self.createInvalidFogUrlLoadBalancer()
+        invalidConsensusUrlBalancer = try Self.createInvalidConsensusUrlLoadBalancer()
+    }
+}
+
+protocol ServiceFixture {
+    var serviceName: String { get }
+    var urlTypeName: String { get }
+    var loadBalancer: MockUrlLoadBalancer { get }
+
+    func callService(expect: XCTestExpectation) throws
+}
+
+extension UrlLoadBalancerFixtures {
+
+    struct Blockchain: ServiceFixture {
+        let serviceName = "Blockchain"
+        let urlTypeName = "Consensus"
+        let seqLoadBalancer: SequentialUrlLoadBalancer<ConsensusUrl>
+        let blockchain: BlockchainConnection
+
+        var loadBalancer: MockUrlLoadBalancer {
+            seqLoadBalancer
+        }
+
+        init() throws {
+            self.seqLoadBalancer = try UrlLoadBalancerFixtures().invalidConsensusUrlBalancer
```

Comment:
This PR adds alot of abstractions that could be hard to GROK, can you collapse these structs inline where the test is called ? Will improve readability and its essentially the test-code so its ok for it to be a little longer at the call-site.

Reply:
This was the same format used for the prior tests - so that's why I repeated this...

Using the fixtures reduced a lot of redundant code - since the creation of the service couldn't be abstracted/generalized, it was wrapped in a fixture when I wrote the integration tests.

You sure you want to change it?

Reply:
de-abstracted tests a bit from the fixture so you can see the individual test logic a bit more clearly